### PR TITLE
Portfolio Project Mobile Menu

### PR DESCRIPTION
**Pull request by nurbolsultanov**
1. Implemented ES Linters ✔️ 
2. Added a script.js file ✔️ 
3. Added the function to the hamburger icon ✔️ 
4. Added the function to the close icon ✔️ 
5. Added style to the mobile menu ✔️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,15 +24,15 @@
         <div class="logo">
             <a href="#">My Logo</a>
         </div>
-        <ul class="navigation-menu">
+        <ul class="navigation-menu" id="nm">
             <li><a class="navigation" href="#works">Portfolio</a></li>
             <li><a class="navigation" href="#aboutme">About</a></li>
             <li><a class="navigation" href="#contact">Contact</a></li>
         </ul>
         <div class="hamburger">
```
Comment (GzimAsani) [inline index.html]: - [x] The menu should be displayed all the way to the bottom and users should not be able to click other buttons behind the menu when menu is open
image

Comment (GzimAsani) [reply index.html]: - [x] Adjust the items and the X as on the design

Yours:
image
Design:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @everinurmind                      :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your hamburger :+1: 

 ## Required Changes 
- [x] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (nurbolsultanov) [reply index.html]: On my end the menu overlay is displayed all the way down the page, can not really understand why it is not on your end.

File: style.css [hunk 2]
```diff
@@ -24,7 +24,8 @@
   }
```
Comment (HAMMAS-SALEEM) [inline style.css:24]: image

**[Optional]**

- [x] Kindly add the necessary commands for running your project in a local machine in these sub-sections :+1: and if any sub section is unnecessary then kindly remove that sub-section :blush:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @everinurmind, @ghreza-crypto 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
:heavy_check_mark: Following GitHub Flow

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**